docs/sphinx/conf.py

```diff
@@ -127,7 +128,7 @@
 help_root = 'http://help.openmicroscopy.org'
 
 # Edit on GitHub prefix
-edit_on_github_prefix = 'formats'
+edit_on_github_prefix = 'ome-model'
```

Comment:
It looks like the edit_on_github_prefix should be  `docs/sphinx/` as currently the links look like:
https://github.com/ome/ome-model/edit/develop/formats/developers/index.rst

as opposed to the desired 
https://github.com/ome/ome-model/edit/develop/docs/sphinx/developers/index.rst